admin_guide/node_problem_detector.adoc

```diff
@@ -0,0 +1,426 @@
+                "source": "kernel"
+        },
+        "logPath": "/host/log/journal",
+        "lookback": "5m",
+        "bufferSize": 10,
+        "source": "kernel-monitor",
+        "conditions": [
+                {
+                        "type": "KernelDeadlock",
+                        "reason": "KernelHasNoDeadlock",
+                        "message": "kernel has no deadlock"
+                }
+        ],
+        "rules": [
+                {
+                        "type": "temporary",
+                        "reason": "OOMKilling",
+                        "pattern": "Kill process \\d+ (.+) score \\d+ or sacrifice child\\nKilled process \\d+ (.+) total-vm:\\d+kB, anon-rss:\\d+kB, file-rss:\\d+kB"
+                },
+                {
+                        "type": "temporary",
+                        "reason": "TaskHung",
+                        "pattern": "task \\S+:\\w+ blocked for more than \\w+ seconds\\."
+                },
+                {
+                        "type": "temporary",
+                        "reason": "UnregisterNetDevice",
+                        "pattern": "unregister_netdevice: waiting for \\w+ to become free. Usage count = \\d+"
+                },
+                {
+                        "type": "temporary",
+                        "reason": "KernelOops",
+                        "pattern": "BUG: unable to handle kernel NULL pointer dereference at .*"
+                },
+                {
+                        "type": "temporary",
+                        "reason": "KernelOops",
+                        "pattern": "divide error: 0000 \\[#\\d+\\] SMP"
+                },
+                {
+                        "type": "permanent",
+                        "condition": "KernelDeadlock",
+                        "reason": "AUFSUmountHung",
+                        "pattern": "task umount\\.aufs:\\w+ blocked for more than \\w+ seconds\\."
+                },
+                {
+                        "type": "permanent",
+                        "condition": "KernelDeadlock",
+                        "reason": "DockerHung",
+                        "pattern": "task docker:\\w+ blocked for more than \\w+ seconds\\."
+                }
+        ]
+    }
+----
+
+. Remove or add any NodeConditions (conditions) or events (rules) as needed. 
+
+. Restart running pods to apply the changes. To restart pods, you can delete all existing pods:
++
+[source,bash]
+----
+# oc delete pods -n openshift-node-problem-detector -l name=node-problem-detector
+----
+
+[[admin-guide-node-problem-detector-run]]
+== Reinstalling the Node Problem Detector
+
+If for any reason, you need to re-create the xref:../dev_guide/daemonsets#dev-guide-daemonsets[daemonset] object, run the following Ansible playbook:
+
+[source,bash]
+----
+ansible-playbook ~/openshift-ansible/playbooks/openshift-node-problem-detector/config.yml
+----
+
+You can also manually create the daemonset object, using the following steps. 
```

Comment:
We either need to 1) take the manual install section out or 2) update it to match the `ds` we create with the installer. My preference is 1 since otherwise we'd need to update this doc for every version to point to the correct NPD image for the product version for the doc. (i.e. origin vs. ocp, 3.9 vs 3.10, etc). If you want to keep it, let me know and I'll send you the `ds` from the installer.